src/access/universal.test.ts

```diff
@@ -1268,3 +1271,113 @@ describe("getGroupAccessAll", () => {
     expect(access).toBeNull();
   });
 });
+
+describe("getAccessFor", () => {
+  it("calls to getAgentAccess with the appropriate parameters", async () => {
+    const universalModule = jest.requireActual("./universal") as {
+      getAgentAccess: () => Promise<Access | null>;
+    };
+    universalModule.getAgentAccess = jest.fn();
+    const options = {
+      fetch: jest.fn().mockResolvedValue({
+        ok: false,
+      } as never) as typeof fetch,
+    };
+    await getAccessFor(
+      "https://some.resource",
+      "agent",
+      "https://some.pod/profile#webid",
+      options
+    );
+    expect(universalModule.getAgentAccess).toHaveBeenCalledWith(
+      "https://some.resource",
+      "https://some.pod/profile#webid",
+      expect.objectContaining(options)
```

Comment:
I think this can now just be `options` again (just like in the other tests), right?

```suggestion
      options
```

Reply:
Indeed 3118821075055cc5f9eafccbb13fb32d6046736c